_ml-commons-plugin/api/model-apis/delete-model.md

```diff
@@ -44,4 +44,15 @@ DELETE /_plugins/_ml/models/MzcIJX8BA7mbufL6DOwl
   "_seq_no" : 27,
   "_primary_term" : 18
 }
-```
\ No newline at end of file
+```
+
+## Safely deleting a model
+Introduced 2.19
+{: .label .label-purple }
+
+To prevent accidental deletion of models actively used by agents, search pipelines, ingest pipelines, or other components, you can enable a safety check. If you try to delete a model that is in use with the safety check enabled, OpenSearch returns an error message. To proceed with deletion:
```

Comment:
```suggestion
To prevent accidental deletion of models in active use by agents, search pipelines, ingest pipelines, or other components, you can enable a safety check. If the safety check is enabled and you attempt to delete a model that is in current use, OpenSearch returns an error message. To proceed with deletion:
```